src/app/modules/lti/pages/lti/lti.component.ts

```diff
@@ -26,7 +26,7 @@ const noChildError = new Error(LTIError.NoChild);
 const loginError = new Error(LTIError.LoginError);
 
 const isRedirectionParam = 'is_redirection';
-const userIdParam = 'user_id';
+const loginIdParam = 'login_id';
```

Comment:
No, the parameter given by the login module is still `user_id`!  It is just that "user_id" is the "loginIdParam" for the frontend.

Reply:
fixed